### DISCUSSION TITLE
How can I disable the floating notification that says Committing...

### DISCUSSION THREAD

**Discussion by maxigaz** [NeogitOrg/neogit#486] [Q&A]
First of all, thank you for this plugin!

I'm using Neogit installed through Packer, with the default settings, with Neovim v0.9.0.
When I bring up the screen to type in a commit message (by staging a few hunks with gitsigns, running `:Neogit commit` and pressing `c`), I immediately see a floating notification on the right that says `Committing...`, even before I type anything, and then it automatically disappears after a few seconds.

While I generally like this kind of notification, I find this particular one redundant and distractive. Is there a way to disable it?

**Comment by CKolkey** [comment]
So, that's coming from [here](https://github.com/TimUntersberger/neogit/blob/master/lua/neogit/popups/commit.lua#L12), and there's not a way to disable just that one. However you _could_ overload [this](https://github.com/TimUntersberger/neogit/blob/master/lua/neogit/lib/notification.lua#L6) function, and just filter out the message you don't want to see, if you're so inclined.
